.github/changelog-processor.py

```diff
@@ -26,21 +26,38 @@
     help="Should a release be made? Prints `1` or `0`.",
     action="store_true"
 )
+group.add_argument(
+    "--print-changelog-from-last-release",
+    dest="changelog_last_release",
+    help="Print the changelog from the last release.",
+    action="store_true"
+)
 
 args = parser.parse_args()
 
 with open(args.changelog, "r") as changelog:
     lines = changelog.readlines()
 
+    changelog_last_release = ""
+    found_last_version = False
+
     # Find the latest version
     for line in lines:
         if not line.startswith("## ["):
+            changelog_last_release += line
             continue
+        elif not found_last_version:
+            changelog_last_release += line
+            found_last_version = True
+            version = line.strip().removeprefix("## [").split("]")[0]
+        else:
+            break
```

Comment:
how about not including a header from the changelog.
can be something like (tested locally) -

```suggestion
    # Find the latest version
    # Collect CHANGELOG from the latest version
    for line in lines:
        if found_last_version and line.startswith("## ["):
            break
        if not found_last_version and line.startswith("## ["):
            found_last_version = True
            version = line.strip().removeprefix("## [").split("]")[0]
        if found_last_version:
            changelog_last_release += line
```

Reply:
probably even better to have that header in the release. just ignore this if you think so (:

Reply:
I would keep it for now and maybe change it in the future if it is not fitting.

Reply:
I can not resolve the conversation for some reason